rbi/stdlib/find.rbi

```diff
@@ -33,8 +33,8 @@ module Find
     params(
       paths: String,
       ignore_error: T::Boolean,
-      blk: T.proc.params(path: String).void
-    ).void
+      blk: T.nilable(T.proc.params(path: String).void)
+    ).returns(T.nilable(Enumerator))
```

Comment:
```suggestion
    ).returns(T.nilable(T::Enumerator[String]))
```

Reply:
It will indeed always be a`String` (file or dir name). I've applied your suggestion.